tests/test_basic.py

```diff
@@ -69,9 +69,9 @@ def test_set_camera_identifier(runtime: Runtime):
     ],
 )
 def test_select_one_of(
-    runtime: Runtime,
-    input: List[str],
-    expected: str,
+        runtime: Runtime,
```

Comment:
did you run black? there's a precommit file if you want to use that

Reply:
I have now!